pandas/core/frame.py

```diff
@@ -1305,6 +1307,11 @@ def to_numpy(self, dtype=None, copy: bool = False) -> np.ndarray:
             another array. Note that ``copy=False`` does not *ensure* that
             ``to_numpy()`` is no-copy. Rather, ``copy=True`` ensure that
             a copy is made, even if not strictly necessary.
+        na_value : Any, optional
+            The value to use for missing values. The default value depends
+            on `dtype` and the type of the array.
```

Comment:
what happens with mixed dataframes?

Reply:
Then the same value is used for all dtypes.

(in principle you could imagine this keyword being a dict to specify a value per column, but let's leave that for later if there is actually a use case / demand for this)